### ISSUE TITLE
Show upload progress on web page

### ISSUE THREAD

**Issue by Bartalog** [Bartalog/cool-maze#339]
For large upload, or slow connection, when QR has already been scanned : 
- every N seconds, send a progress ratio from mobile to target (via backend)

This makes sense only if the bottleneck is the mobile connection, not the computer connection.

**Comment by Bartalog** [comment]
While it is nice to have some progress feedback on the computer, this one may be slightly misleading because it's only a Mobile->GCS progress indicator. It says nothing about the subsequent GCS->target download, which (for single resource) starts only after the upload is 100% complete.
